examples/add_image.py

```diff
@@ -3,16 +3,16 @@
 properties
 """
 
-from skimage import data
-from skimage.color import rgb2gray
+from imageio import imread
 from napari import ViewerApp
 from napari.util import app_context
 
 
+astro = imread('imageio:astronaut.png').mean(axis=2) / 255
```

Comment:
Oh god, I take back the support I expressed in the meeting, this is just terrible LOL. I think this refactor should include some test data. We can use [pooch](https://www.leouieda.com/blog/introducing-pooch.html) to use some fluorescence or EM data, and avoid this kind of "unnatural" loading.

Reply:
@jni I don't have super strong opinions on this one, but can you explain what is "unnatural" about this loading and how pooch is different (from a loading standpoint). Is it the dynamic downloading / caching of the data? I feel like pooch does that too.

I get that pooch would allow us to use our own more scientifically interesting datasets - and maybe that would be sufficient justification for not doing this refactor, but I sense you have other objections!

Reply:
> can you explain what is "unnatural" about this loading and how pooch is different (from a loading standpoint)

With pooch you can use a different process/function (we can do this explicitly or with some magic — a file that would always be run when the examples are run) and then get *an actual filename with a real path in the filesystem*, which would match how people would normally load a file. This `imageio:` gives me the heebie-jeebies. =P

> pooch would allow us to use our own more scientifically interesting datasets

yeah that's another advantage. We could use EM data for grayscale examples and fluorescence data for multichannel ones.

Reply:
Ah so it's really about `imageio:astronaut.png` vs some more conventional path like `examples/astronaut.png`. I guess I don't have quite the aversion to the first one, but don't see this as so critical either. Curious how others feel. @kne42 @kevinyamauchi